sound/soc/intel/boards/sof_sdw.c

```diff
@@ -807,7 +823,7 @@ static int create_sdw_dailink(struct device *dev, int *be_index,
 			      playback, capture,
 			      cpus + *cpu_id, cpu_dai_num,
 			      codecs, codec_num,
-			      NULL, &sdw_ops);
+			      NULL, codec_info_list[codec_index].ops);
```

Comment:
@RanderWang this change is really hard to follow. Can youplease explain this in the commit message? I cant quite understand what this does

Reply:
thanks, I will update it when we get a conclusion